tests/data_context/cloud_data_context/test_datasource_crud.py

```diff
@@ -124,16 +125,12 @@ def mocked_post_response(*args, **kwargs):
         if save_changes:
             # Make sure the id was populated correctly into the created datasource object and config
             assert stored_datasource.id == datasource_id
-            assert retrieved_datasource.id == datasource_id
```

Comment:
Since we are mocking the retrieved datasource this test would just be checking the fixture accuracy. Removing to make it more clear what we are testing.